dashboard/config.yml

```diff
@@ -4,4 +4,15 @@ server:
     port: 8000
   adminConnectors:
   - type: http
-    port: 8001
\ No newline at end of file
+    port: 8001
+deployer:
+  host: ${DEPLOYER_HOST}
```

Comment:
The only problem to using Brooklyn yaml templating is that it makes the Dashboard a bit more Brooklyn dependent (which is not a big problem!). Maybe we could support two templates or something similar.

However, this template is OK for building with any kind of distribution ATM. I'll add this feature when the brooklyn dashboard entity is ready, which will be soon.